INSTALL.md

```diff
@@ -37,29 +45,32 @@ Unless you are confident with editing `.bat` files and altering the system
 PATH variable, please use the recommended instructions and directory names
 below. 
 
-_Note that you must do step 6, even if you have done it previously, as
+_Note that you must do step 8, even if you have done it previously, as
 the version of Guiprep you are installing may require additional Perl modules
 to any previous versions._
 
 1. Download [Strawberry Perl](http://strawberryperl.com/).
 2. Double click the downloaded file to install Strawberry Perl. It is
    recommended that you install in the default folder `c:\Strawberry`.
-3. Download the latest release (the zip file) from the
-   [Guiprep releases](https://github.com/DistributedProofreaders/guiprep/releases) page.
-4. Unzip guiprep zip file to some location on your computer (double
+3. Download [ActiveTcl](https://www.activestate.com/products/tcl/).
```

Comment:
@windymilla - how many of our users are on non-64-bit Windows boxes? I ask because it appears that you have to pay for the 32-bit ActiveTcl https://www.activestate.com/products/tcl/#how-can-i-get-the-32-bit-tcl-download

Reply:
I don't know the answer to that. I think it would be sufficiently few, and guiprep/guiguts both sufficiently stable, that it would be OK if they had to remain on their current version (maybe occasional bug fix in the pre-tcl version if really serious). 
Also, I think it was necessary for me to make a (free) account to get Tcl (don't remember for sure) so some might refuse to do that, in which case they would remain on the old version too.

Reply:
Good call. Another reason to try to see if the open source version works fine. Feels weird making people make an account to install this software.

Reply:
Fixed with ad76a9b.